src/cpp/core/ProgramOptions.cpp

```diff
@@ -54,21 +54,42 @@ bool validateOptionsProvided(const variables_map& vm,
 
 }
 
-void reportError(const Error& error, const ErrorLocation& location)
+void reportError(const Error& error, const ErrorLocation& location, bool forceStderr)
 {
    std::string description = error.getProperty("description");
    if (core::system::stderrIsTerminal() && !description.empty())
+   {
       std::cerr << description << std::endl;
+   }
+   else
+   {
+      // in some cases, we may need to force stderr to be written
+      // for example, during installation on RedHat systems, stderr
+      // is not properly hooked up to a terminal when checking configuration during post install scripts
+      // which would cause error output to go only to syslog and be hidden from view during install
+      if (forceStderr && !description.empty())
+         std::cerr << description << std::endl;
+
+      core::log::logError(error, location);
```

Comment:
Prior to this change, `reportError` would _always_ log the error -- now many errors won't be logged if stderr is a terminal. Is that intentional?

Reply:
@kfeinauer, since you're OOO I made this change --- https://github.com/rstudio/rstudio/pull/8635/commits/f54f85170c024ebf0521a4ffd0e8e292e4e05d88. LMK if that doesn't look correct.